doc/releases/nautilus.rst

```diff
@@ -216,13 +216,24 @@ Instructions
         "ceph version 14.2.0 (...) nautilus (stable)": 22,
      }
 
-#. Scan for any OSDs deployed with the old ceph-disk tool to ensure
-   that ceph-volume can activate them after a host reboot.  On each
-   host containing OSDs,::
+#. If there are any OSDs in the cluster deployed with ceph-disk (e.g.,
+   almost any OSDs that were created before the Mimic release), you
+   need to tell ceph-volume to adopt responsibility so they can be
+   activated after a host reboot.  On each host containing OSDs,::
 
      # ceph-volume simple scan
      # ceph-volume simple activate --all
 
+   We recommend that each OSD host be rebooted following this step to
```

Comment:
Should we recommend setting `noout` for that reboot to avoid uncessary recovery?

Reply:
noout is already set for the full upgrade procedure